types/office-js-preview/index.d.ts

```diff
@@ -69907,6 +69821,10 @@ declare namespace Word {
         */
         toJSON(): Word.Interfaces.DocumentData;
     }
+    class DocumentCustom {
```

Comment:
Hmm. Similar problems to the other new pair of class and interface: 
1. There are no descriptions
2. No members of the class/interface.
3. Odd syntax with comma in `extends` declaration.  I can't find any example of this syntax online.